src/devices/Pn532/ParametersFlags.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Pn532
+{
+    /// <summary>
+    /// Security Access Module parameters
+    /// </summary>
+    [Flags]
+    public enum ParametersFlags
```

Comment:
should this be called `SecurityAccessModuleParameters` then the comment could be removed

Reply:
It's acutally a mix. And the original name from the documentation is not the best choice from my point of view. For example, the RemovePrePostAmble has nothing related to the security. I'll see what I can do to bring clarity and see what I can rename in the main function as well as in this enum.

Reply:
I feel it's still better than current name. I'd go with just renaming this according to spec and then extend from there later on (not super high pri though).